src/sql/workbench/services/restore/browser/restoreDialog.ts

```diff
@@ -358,14 +424,15 @@ export class RestoreDialog extends Modal {
 					c.appendChild(optionsContentElement);
 				}
 			}
-		});
+		};
+		this._panel.pushTab(this._optionsTab);
 
 		this._panel.onTabChange(c => {
```

Comment:
This should be registered (same as the components if you could fix that up while you're here for all other event handlers that'd be great)